src/.gitignore

```diff
@@ -8,3 +8,7 @@ typechain-types
 # Hardhat files
 cache
 artifacts
+
+# Forge files
+cache_forge
```

Comment:
In our stability pool compounder project ([https://github.com/Byte-Masons/stability-pool-compounder](url) we use forge-cache. Are you sure it is cache_forge?

Reply:
yes, it is a default cache directory and I was not overriding it in `foundry.toml` file

Reply:
regarding `remappings.txt` file - it seems it is not needed at this point. Currently, this template only loads `Test.sol` contract from `forge` and test works even though I do not specify the full path (`forge-std/Test.sol` instead of `lib/forge-std/src/Test.sol`) - so I assume it does some implicit mappings